src/Jackalope/ObjectManager.php

```diff
@@ -829,6 +831,13 @@ public function save()
                         if ($node->needsChildReordering()) {
                             $this->transport->reorderChildren($node);
                         }
+                        // add versioning properties
+                        foreach($node->getMixinNodeTypes() as $mixinType) {
```

Comment:
Wouldn't be necessary if the `NodeInterface` contains a `hasMixin`-method. Can we add this to the Interface, or are we strictly following the specification here?